README.md

```diff
@@ -18,4 +18,16 @@ Start Jupyter notebook.
 
 ```
 $> doit notebook
+```
+
+## Dash app
+Make sure you have the following packages installed
+* dash
```

Comment:
Very minor concern, but I think it suffices to mention to `pip install -r requirements.txt` first.